packages/graphql-eslint-rules/src/utils.ts

```diff
@@ -7,12 +7,16 @@ export function checkDirForPkg(dir: string): string | null {
 
   try {
     stat = fs.statSync(pathname, { throwIfNoEntry: false } as StatOptions);
```

Comment:
Probably don’t need this option at all if we’re going to start an exception scope anyways, right?